bot/cogs/moderation/silence.py

```diff
@@ -61,7 +32,7 @@ def __init__(self, bot: Bot):
         self._get_instance_var_task = self.bot.loop.create_task(
             self._get_instance_vars())
         self._get_instance_vars_event = asyncio.Event()
-        self.scheduler = UnsilenceScheduler(bot)
+        super().__init__()
```

Comment:
It is a convention to use `super().__init__()` at the beginning unless there is a good reason to.